tests/e2e/models/mnist_train_test/generate_test_data.py

```diff
@@ -0,0 +1,219 @@
+# Copyright 2023 Google LLC
```

Comment:
Where is this file copied from? I'm not seeing an obviously matching one in Jax.

Reply:
My bad. This is a mistake from my copy-paste driven development. Should be fixed now.